common/annotations/en.xml

```diff
@@ -821,9 +821,9 @@ annotations.
 		<annotation cp="🫨">earthquake | face | shaking | shock | vibrate</annotation> <!-- 1FAE8 -->
 		<annotation cp="🫨" type="tts">shaking face</annotation>
 		<!-- Generated lines from Emoji Data v15.1, using GenerateCldrData.java -->
-		<annotation cp="🙂‍↔">head shaking horizontally | no, shake</annotation> <!-- 1F642 200D 2194 -->
+		<annotation cp="🙂‍↔">head shaking horizontally | no | shake</annotation> <!-- 1F642 200D 2194 -->
 		<annotation cp="🙂‍↔" type="tts">head shaking horizontally</annotation>
-		<annotation cp="🙂‍↕">head shaking vertically | nod, yes</annotation> <!-- 1F642 200D 2195 -->
+		<annotation cp="🙂‍↕">head shaking vertically | nod | yes</annotation> <!-- 1F642 200D 2195 -->
```

Comment:
@AEApple I'm not so sure.  There's "no, shake" and "nod, yes" but i think these should have been `no | shake` and `nod | yes`.